kafka/pom.xml

```diff
@@ -46,6 +46,7 @@
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <mycila-license.version>3.0</mycila-license.version>
         <strimzi.testcontainers.version>0.20.1</strimzi.testcontainers.version>
+        <kafka-oauth-client.version>0.7.2</kafka-oauth-client.version>
```

Comment:
I still wonder whether we should ask Quarkus to manage `io.strimzi:kafka-oauth-client`

Reply:
+1